src/cmd/root.go

```diff
@@ -382,12 +382,12 @@ func validateSubIDFile(path string) (bool, error) {
 	scanner := bufio.NewScanner(file)
 	scanner.Split(bufio.ScanLines)
 
-	prefix := currentUser.Username + ":"
-
 	for scanner.Scan() {
 		line := scanner.Text()
-		if strings.HasPrefix(line, prefix) {
-			return true, nil
+		for _, prefix := range [2]string{currentUser.Username + ":", currentUser.Uid + ":"} {
```

Comment:
Nitpick: will be slightly more readable to split the slice of prefixes into a separate line.